scripts/core/EXT_Exp_GroupAlgorithmMemory.rst

```diff
@@ -0,0 +1,34 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+%>
+:orphan:
+
+.. _ZE_experimental_device_group_algorithm_memory_properties:
+
+========================================================
+Group Algorithm Memory Properties Experimental Extension
```

Comment:
are we adding some algorithms to Level Zero ? 
Is there somewhere more data on the request ?

Reply:
thanks @MichalMrozek . Sent email to you on this.